src/components/NewRelease/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import useTranslation from 'next-translate/useTranslation'
+import Trans from 'next-translate/Trans'
+import Link from 'next/link'
+
+interface NewReleaseItem {
+  leftContent?: boolean
+}
+
+const NewReleaseItem = ({ leftContent }: NewReleaseItem) => {
+  const { t } = useTranslation('home')
+  const Header = (props) => <h2 className="new-release-item__header" {...props} />
+
+  const renderIntroductionHeader = () => (
+    <Trans
+      i18nKey={t('newReleaseItemSectionHeader')}
+      components={[<Header />, <br />]}
+    />
+  )
```

Comment:
Since we don't pass any props to this `h2` element, I think we can rewrite it this way. What do you think? 🧐

```suggestion
  const header = [<h2 className="new-release-item__header" />, <br />]

  const renderIntroductionHeader = () => (
    <Trans i18nKey={t('newReleaseItemSectionHeader')} components={header} />
  )
```